spec/components/image_card_spec.rb

```diff
@@ -123,4 +123,9 @@ def component_name
     render_component(href: "#", lang: "cy")
     assert_select ".gem-c-image-card[lang='cy']"
   end
+
+  it "applies lazy loading attribute when lazy is specified" do
```

Comment:
can we update other tests to reflect that `"auto"` is applied by default?

Reply:
Course, added